docs/subscriptions.md

```diff
@@ -147,7 +147,7 @@ import { pubSub } from "./pubsub";
 class SampleResolver {
   // ...
   @Mutation(returns => Boolean)
-  async addNewComment(@Arg("comment") input: CommentInput, @PubSub() pubSub: PubSubEngine) {
+  async addNewComment(@Arg("comment") input: CommentInput) {
```

Comment:
Yeah I forgot to remove this line in that PR. Thanks for spotting this!

Reply:
PR is ok now :)